packages/libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -908,6 +908,19 @@ export class ChatsApi
         } else if (data.rpc.method === 'chat.blast') {
           const userId = data.metadata.receiverUserId
           await this.upgradeBlasts(userId)
+          this.eventEmitter.emit('blast', {
+            audience: data.rpc.params.audience,
+            audienceContentType: data.rpc.params.audience_content_type,
+            audienceContentId: data.rpc.params.audience_content_id,
+            message: {
+              message_id: data.rpc.params.blast_id,
+              message: data.rpc.params.message,
+              sender_user_id: userId,
+              created_at: data.metadata.timestamp,
+              reactions: [],
+              is_plaintext: true
```

Comment:
Tbh don't think we need this? Same w/ above in `emit('message')` - both should be returning plain text

Reply:
hm you're probably right. follow-up PR thing.